src/client/ContentAccess.js

```diff
@@ -934,8 +934,8 @@ exports.ContentObjectMetadata = async function({
     // For a 404 error, check if error was due to write token not found
     const errQwtoken = objectPath.get(error.body, "errors.0.cause.cause.cause.qwtoken");
     if(errQwtoken) {
-      // if so, re-throw rather than suppress error
-      throw error;
+      // if so, throw 'write token not found' error
+      throw Error(`Write token ${writeToken} not found  - draft already finalized or wrong node queried?`);
```

Comment:
I would remove 'or wrong node queried' as we currently do redirect to the correct node in qfab API.
There is a specific caveat for when the node is unavailable - that should never happen in regular operation but when it does, I believe the HTTP error we'd be getting on the client side is 'too many redirects'.
Maybe for this PR we can say draft already finalized or node unavailable (and then we can refine as we have more info)

Reply:
ok, changing message to "Write token X not found  - draft already finalized or node unavailable?"